### DISCUSSION TITLE
[Feature Request] Support Open API

### DISCUSSION THREAD

**Discussion by ruscon** [spatie/laravel-route-attributes#102] [Ideas]
[This library](https://github.com/zircote/swagger-php) is very often used.

Maybe add a bridge?

**Comment by kdevan** [comment]
Yeah, I have controllers that looks like this right now with duplicate attributes to support both. I'm using that same library. I planned to do a sort of bridge type thing when I can prioritize it so just adding my vote here.

![Screenshot from 2023-07-23 02-21-26](https://github.com/spatie/laravel-route-attributes/assets/1412685/1ce4149d-5403-401b-bdc5-b200083b79e3)
